lib/web/jquery.js

```diff
@@ -42,7 +42,7 @@
 // Pass this if window is not defined yet
 }(typeof window !== "undefined" ? window : this, function( window, noGlobal ) {
 
-// Support: Firefox 18+
+// SupportMessage: Firefox 18+
```

Comment:
@holmatro90 Must be VERY CAREFUL with refactoring and must review changed before committing them. It is bad practice to change core files!